### PR TITLE
Address typos

### PR DESCRIPTION
**Pull request by hguthrie**
The purpose of this pull request is to quickly address some random typos and formatting issues. This PR does not improve the content meaning.

The cloud topic `user-admin.md` may need a second look. Part of the sentence was missing and I improvised.

### CODE REVIEW

File: src/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Customer sections was designed to cache private data in browser storage. This me
 The are some exception cases:
 
 -  Store and website switching, after any of these action customer section `cart` will be updated.
--  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passe section `cart` will be updated.
+-  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passes section `cart` will be updated.
```
Comment (BarnyShergold) [inline src/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.md:70]:
```suggestion
-  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passes, section `cart` will be updated.
```

**Review by BarnyShergold — APPROVED** [review]
One tiny, tiny suggestion so will approve

**Review by dobooth — APPROVED** [review]

File: src/cloud/project/user-admin.md [hunk 2]
```diff
@@ -259,7 +259,7 @@ To enable TFA on your {{site.data.var.ece}} user account:
 
    -  Log in with your username and password.
 
-   -  When prompted, enter the two-factor authentication code for the magento.cloufrom the authenticator application on your mobile device
+   -  When prompted, enter the two-factor authentication code for `magento.cloud` from the authenticator application on your mobile device.
```
Comment (meker12) [inline src/cloud/project/user-admin.md:262]:
```suggestion
   -  When prompted, enter the two-factor authentication code for the `accounts.magento.cloud` entry from the authenticator application on your mobile device.
```

Comment (hguthrie) [reply src/cloud/project/user-admin.md:262]: thanks!

File: src/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ Customer sections was designed to cache private data in browser storage. This me
 The are some exception cases:
 
 -  Store and website switching, after any of these action customer section `cart` will be updated.
--  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passe section `cart` will be updated.
+-  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After scheduled time passes, section `cart` will be updated.
```
Comment (meker12) [inline src/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.md:70]:
```suggestion
-  Customer cart lifetime option `section_data_lifetime` which is 60 minutes by default. After the scheduled time passes, the `cart` section updates.
```

**Review by meker12 — APPROVED** [review]
A few suggestions

**Comment by hguthrie** [comment]
running tests

**Pull request merged**